be/src/common/config.h

```diff
@@ -353,6 +353,7 @@ CONF_mInt32(stream_load_record_batch_size, "50");
 CONF_Int32(stream_load_record_expire_time_secs, "28800");
 // time interval to clean expired stream load records
 CONF_mInt64(clean_stream_load_record_interval_secs, "1800");
+CONF_mBool(disable_stream_load_2pc, "false");
```

Comment:
I think we can set default value to true. And set it true after we complete the flink connector and everything is ok,

Reply:
> I think we can set default value to true. And set it true after we complete the flink connector and everything is ok,

OK.